lib/Whateverable.pm6

```diff
@@ -108,6 +108,29 @@ multi method irc-to-me(Message $msg where .text ~~ /:i^ uptime \s* $/) {
         ~ “implementing {.language_name} {.language_version}.”
      })
 }
+#| Get the oldest commit DateTime for $*PROGRAM. Needs $*CWD to point to the whateverable git repository.
+sub term:<birthday> {
+    my $dates = run :out, ‘git’, ‘log’, ‘--follow’, ‘--date-order’, ‘--format=%aI’, $*PROGRAM.relative;
+    try { $dates.out.lines(:close).tail.&{ DateTime.new($_) }.Date } \
+        or fail ‘no birthday data available’
+}
+#| Proper answer to “how old are you?”
+sub my-age is export {
+    with birthday {
+        “I was created on $_”
+    }
+    else {
```

Comment:
Why is this needed? Bots automatically gist uncaught exceptions, there's no need. You can of course `grumble ‘something went wrong’` but I don't see any good reason. In this case exception is totally fine, although I'm a bit confused as to when it'll trigger given that ‘no birthday data available’ message is there? Maybe wrap the whole thing in a try/catch and just output ‘no birthday data available no matter what goes wrong? The feature is more like an easter egg, so I'm not too concerned about debugging.

Also, change it to ‘I don't remember’, I think that's funnier.